site/src/pages/CreateWorkspacePage/CreateWorkspacePageView.tsx

```diff
@@ -110,6 +113,11 @@ export const CreateWorkspacePageView: FC<React.PropsWithChildren<CreateWorkspace
     )
   }
 
+  const canSubmit =
+    props.quota && props.quota.user_workspace_limit > 0
+      ? props.quota.user_workspace_count < props.quota.user_workspace_limit
+      : true
```

Comment:
I _think_ we can simplify if you want

```suggestion
    props.quota && props.quota.user_workspace_limit > 0 && props.quota.user_workspace_count < props.quota.user_workspace_limit
```